tests/utils/stl/test/params.py

```diff
@@ -36,6 +36,23 @@ def pretty(self, config, litParams):
     return 'exclude run.pl tags {}'.format(str(self._taglist))
 
 
+def beNice(prio: str) -> list[ConfigAction]:
+  """
+  Set the process priority to run tests with.
+  """
+  try:
+    import psutil
+    priority_map = {
+      'normal': psutil.NORMAL_PRIORITY_CLASS,
+      'low': psutil.BELOW_NORMAL_PRIORITY_CLASS,
+      'idle': psutil.IDLE_PRIORITY_CLASS,
+    }
+    psutil.Process().nice(priority_map[prio])
+  except ImportError:
+    pass

```

Comment:
Instead of silently ignoring the option (including the default of idle priority), I think it would be better to print a note explaining that psutil's absence will cause this setting to be ignored. This will be helpful even for long-standing contributors/maintainers who might occasionally run tests on a fresh system. Phrasing it as a "note" rather than a "warning" seems ideal, since this is ignorable.